### ISSUE TITLE
Add a way to override the token cookie name

### ISSUE THREAD

**Issue by elghazal-a** [nitnelave/lldap#400]
How can we override the cookie token name? I wanted to renamed for example to `lldap_token` to avoid any collision with parent domain

**Comment by nitnelave** [comment]
Yeah, actually #514 was not necessary, since the cookie is not at all accessed on the front-end (it's http only). We only need some server-side setting for this, so it's even easier than anticipated.
